tools/win/generate_debug_symbols.py

```diff
@@ -59,6 +59,13 @@ async def ProcessBinary(semaphore, options, binary):
             elapsed = datetime.utcnow() - start_time
             print(f'Completed generating symbols for {binary}: elapsed time '
                   f'{elapsed.total_seconds()} seconds')
+
+        if options.platform_symbols_dir:
```

Comment:
I'm not quite following what's going on here. Why are we copying to a different directory?

Reply:
this seems especially odd because we're not generating anything different with this change, we're just putting the output in a different place

Reply:
can we add a comment here to explain that this is copying for the standalone native symbols archive?

Reply:
The comment is added